### PR TITLE
feat(cellguide): upload json script

### PR DESCRIPTION
**Pull request by kaloster**
## Reason for Change

- #6247

This is a script to upload cellguide descriptions using the `/cellguide/v1/upload` endpoint.

It requires a csv file to be present in the `scripts` directory with the following columns:
`Cell Ontology ID, Final version (QC'd), Supporting reference 1, Supporting reference 2, Supporting reference 3, 
Supporting reference 4, Supporting reference 5`

The script will convert the csv to json and post each row to the endpoint.

## Testing steps

Step 1 - Login: `python upload_cellguide_descriptions.py --deployment rdev login --api_key <api_key>`
Step 2 - Upload: `python upload_cellguide_descriptions.py --deployment rdev upload --path cl_descriptions.csv`

## Notes for Reviewer
- rdev option is there for testing purposes - once reviewed and approved will be removed before merging.

### CODE REVIEW

File: scripts/upload_cellguide_descriptions.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+@click.group()
+@click.option("--deployment", default="test", show_default=True, help="The name of the deployment to target")
+@click.pass_context
+def cli(ctx, deployment):
+    """
+    Set deployment stage and api base url
+    """
+    os.environ["DEPLOYMENT_STAGE"] = deployment
+    api_base = {
+        "rdev": "https://pr-6426-backend.rdev.single-cell.czi.technology",
```
Comment (kaloster) [inline scripts/upload_cellguide_descriptions.py:30]: rdev option is here for testing purposes - once reviewed and approved this line will be removed before merging.

File: scripts/upload_cellguide_descriptions.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    return json.dumps(json_row)
+
+
+def format_references(csv_row):
+    """
+    Format references list
+    """
+    references = []
+    for i in range(1, 6):
```
Comment (atarashansky) [inline scripts/upload_cellguide_descriptions.py:93]: My memory is murky so I might not be remembering the spec properly - are there known to be only a maximum of 5 supporting references?

**Review by atarashansky — APPROVED** [review]
Left one small question, but otherwise this looks great! I'm trusting that you've tested this in rdev already..

File: scripts/upload_cellguide_descriptions.py (hunk 2 above)
Comment (kaloster) [reply scripts/upload_cellguide_descriptions.py:93]: yes, it's up to not including the upper limit when range is defined

**Pull request merged**